CONTRIBUTING.md

```diff
@@ -42,70 +42,22 @@ Merging Pull Requests (for project contributors with write access)
 
 ### Pre-requisites
 
-This project contains Git submodules which are required to build. After cloning this repo, run `git submodule update
---init --recursive`.
-
-### CMake
-CMake version 3.12 or higher is required to build these libraries. Download and install CMake from the project's
+#### CMake
+CMake version 3.15 or higher is required to build these libraries. Download and install CMake from the project's
 [website](https://cmake.org/download/).
 
-### Vcpkg
-Vcpkg is required to download project dependencies. To get started, first clone vcpkg to a location on your system and
-run the bootstrapping script.
+#### Vcpkg
+Vcpkg can be used to have the Azure for SDK for CPP dependencies installed into a specific folder in the system instead of globally installing them.
+Follow [vcpkg install guide](https://github.com/microsoft/vcpkg) to have next dependencies installed
+
+- curl
+- libxml2
 
 ```sh
-git clone https://github.com/Microsoft/vcpkg.git
-cd vcpkg # Keep note of the location of this directory for the next step
-Windows> .\bootstrap-vcpkg.bat
-Linux/macOS:~/$ ./bootstrap-vcpkg.sh
+./vcpkg install curl libxml2
 ```
 
-On macOS, this command may fail if your version of the C++ toolchain is not new enough to support vcpkg. To resolve
-this, vcpkg recommends to install `gcc@6` from Homebrew (`brew install gcc@6`), then re-run the bootstrapping script.
-
-Next, define the `VCPKG_ROOT` environment variable and add the `vcpkg` command to your path. You will probably want to
-persist these changes, so it's recommended to add/edit them via the Windows "System Properties" control panel, or via
-your `.profile` file on Linux/macOS.
-
-> **Windows**
-> ```bat
-> set VCPKG_ROOT=C:\path\to\vcpkg
-> set PATH=%PATH%;%VCPKG_ROOT%
-> ```
->
-> **Linux/macOS**
-> ```sh
-> export VCPKG_ROOT=/path/to/vcpkg
-> export PATH=$PATH:$VCPKG_ROOT
-> ```
-
-Finally, install the project dependencies with vcpkg.
-
-> **Windows**
-> ```bat
-> set VCPKG_DEFAULT_TRIPLET=x64-windows-static
-> vcpkg install curl[winssl]
-> ```
->
-> **Linux/macOS**
-> ```sh
-> vcpkg install curl[ssl]
-> ```
-
-### Development headers (Linux/macOS)
-On Linux/macOS the development headers for OpenSSL 1.1 must be installed to a location where CMake can find them.
-For Ubuntu 18.04 and up, you can install them directly from the main Ubuntu repository with `apt-get`. For macOS, you
-can install them with Homebrew.
-
-> **Linux (Ubuntu 18.04 and up)**
-> ```sh
-> sudo apt-get install libssl-dev
-> ```
->
-> **macOS**
-> ```sh
-> brew install openssl@1.1
-> ```
+When using VCPKG, make sure to define the `VCPKG_ROOT` environment variable to VCPKG folder before using `CMake`. CMake will pull dependencies from VCPKG when this is set, otherwise, dependencies need to be globally installed. 
```

Comment:
```suggestion
When using VCPKG, make sure to define the `VCPKG_ROOT` environment variable to the VCPKG Git repository folder before using `CMake`. CMake will pull dependencies from VCPKG when this is set, otherwise, dependencies will be installed globally.
```

Do we need to also set the triplet environment variable? We should mention that since it is referred to in `Building the project`.

Reply:
it's not required to link VCPKG to CMake.  Triplet is VCPKG related only. It defines what VCPKG builds for you.
CMake won't use this for anything

Reply:
I don't understand. We ask the reader to set the two environment variables later on (and say "as described above"). And yet, we don't describe it.

https://github.com/Azure/azure-sdk-for-cpp/blob/b78884438d221a711408dee9bec77e6c1f3a33f4/CONTRIBUTING.md#building-the-project
![image](https://user-images.githubusercontent.com/6527137/95640235-a06b9a80-0a50-11eb-9443-70321223b1f8.png)

Reply:
Got it. Let me remove the triplet reference from below, that's VCPKG specific

Reply:
So, just to clarify, do we **not** need the vcpkg triplet when building the C++ repo?

Why are we removing them from the docs? It's used in CI and the CMakeList file:
https://github.com/Azure/azure-sdk-for-cpp/blob/cd2a8a3812725c3524e5d5fd1ab49b9cb08e0890/CMakeLists.txt#L33-L35
https://github.com/Azure/azure-sdk-for-cpp/blob/cd2a8a3812725c3524e5d5fd1ab49b9cb08e0890/eng/pipelines/templates/jobs/archetype-sdk-client.yml#L15

Reply:
@vhvb1989 - please close on this.

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/754